bigchaindb/upsert_validator/validator_election.py

```diff
@@ -0,0 +1,140 @@
+from bigchaindb.common.exceptions import (InvalidSignature,
+                                          DuplicateTransaction)
+from bigchaindb.tendermint.utils import key_from_base64
+from bigchaindb.common.crypto import (public_key_from_ed25519_key)
+from bigchaindb.common.transaction import Transaction, Input, Output
+from bigchaindb.common.schema import (_load_schema,
+                                      _validate_schema,
+                                      TX_SCHEMA_VERSION,
+                                      TX_SCHEMA_COMMON,
+                                      TX_SCHEMA_CREATE)
+
+
+_, TX_SCHEMA_VALIDATOR_ELECTION = _load_schema('transaction_validator_election_' +
+                                               TX_SCHEMA_VERSION, __file__)
+
+
+class ValidatorElection(Transaction):
```

Comment:
Looks like the `unspent_outputs`, `inputs_valid`, and `get_asset_id` methods are broken now.

Reply:
Why?

Reply:
These functions expect `self.operation` to be one of `Transaction.CREATE, Transaction.TRANSFER`, what is not the case for `ValidatorElection`.